project/Build.scala

```diff
@@ -141,7 +141,7 @@ object Build {
   val previousBinaryCrossVersion = CrossVersion.binaryWith("sjs1_", "")
 
   val scalaVersionsUsedForPublishing: Set[String] =
-    Set("2.11.12", "2.12.10", "2.13.1")
+    Set("2.10.7", "2.11.12", "2.12.11", "2.13.2")
```

Comment:
2.10.x here looks wrong.

Reply:
Indeed, fixed.

(also updated a 2.13.1 to 2.13.2 in the sbt plugin tests to fix the CI)